spec/integration/client_side_encryption/explicit_queryable_encryption_spec.rb

```diff
@@ -5,7 +5,7 @@
 
 describe 'Explicit Queryable Encryption' do
   require_libmongocrypt
-  min_server_version '6.0'
+  min_server_version '6.0.0-rc8'
```

Comment:
Honestly I would just keep it at `6.0`. Is your revision to version checks handling the `-rc8`?

Reply:
Apparently, it will not run on 6.0.0-rc8 if we specify just `6.0`
```ruby
Gem::Version.new('6.0.0-rc8') < Gem::Version.new('6.0')
# => true
```

Reply:
I see, that is good then.